adaptors/collections.md

```diff
@@ -4,18 +4,18 @@ title: Collections Adaptor
 
 ## Collections Overview
 
-The Collections API provides access to a secure key/value store on the OpenFn
-Platform. It is designed for high performance over a large volume of data.
+The Collections API provides access to a secure key/value store that allows
+users to store, update and reuse data across workflows in their OpenFn projects.
+It is designed for high performance over a large volume of data.
 
-Collections are secure, private datastores which are visible only to Workflows
-within a particular OpenFn Project. They can be created, managed and destroyed
-from the OpenFn Admin page.
+Collections are secure, private datastores which are only accessible to
+Workflows within a particular OpenFn Project. They can be created, managed and
+destroyed from the OpenFn Admin page.
 
 When running in the CLI, a Personal Access Token can be used to get access to
 the collection (generated from the app at /profile/tokens).
 
-See the [Collections](/documentation/build/collections) Platform Docs to learn
-more about Collections.
+Learn more about collections and common use cases [here](/documentation/build/collections).
```

Comment:
Hey, in formal technical documentation, I don't think it's appropriate to use ` here` as link text. What is here? Where does the link go? It's needlessly mysterious: better to clearly signpost what we're linking to. 

It's usually better to use the page title (even if it might drift out of date over time). But since this page is also called Collections it's probably better to maybe say `Collections Documentation` or even `Main Collections Documentation`

Reply:
uhmm.. will review and push again. Thanks